### PR TITLE
Sigmoid<Add sigmoid function to mindspore frontend> #17954

### PR DESCRIPTION
**Pull request by SHUBHAPRIYA95**

### CODE REVIEW

**Comment by SHUBHAPRIYA95** [comment]
I have tried added [Sigmoid<Add sigmoid function to mindspore frontend> #17954] as an assignment task.

**Review by Ishticode — CHANGES_REQUESTED** [review]
Hi @SHUBHAPRIYA95 
Thank you for the PR. 
I have a few requests.
1) We should just use `ivy.sigmoid` instead of the composition as ivy has implemented this function already for all backends.
2) We don't need docstring in frontend implementations
3) Could you add a test too which we can comment out for now
Feel free to reach out if something is unclear. Thank you very much 😁

**Comment by SHUBHAPRIYA95** [comment]
Hi@Ishticode, 
Thank you for suggestion. Based on suggestion I need to use sigmoid.ivy and write some test cases also right. Could you confirm it that i understand right.

**Comment by SHUBHAPRIYA95** [comment]
Hi@Ishticode, 
By mistake I did re-request review so no need to review that once I will do changes then go for review. Thanks once again!

**Comment by SHUBHAPRIYA95** [comment]
Hi @Ishticode ,
I have done changes in code based on suggestion given.Please review it.

**Review by Ishticode — CHANGES_REQUESTED** [review]
Hey. Thanks for the review request and thank you for the updates. 

The test cases are not to be added manually nor use print statement. Please read our guide here for [Ivy frontend tests](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html). Let me know if something does not make sense and feel free to req a review when updated. Thank you 🙂

**Comment by SHUBHAPRIYA95** [comment]
Hi,@Ishticode,
I was solving the issue i got confused in Ivy frontend test there are jax,numpy,pytorch, tensorflow implementation are there.I want to ask you that should i write sigmoid function test cases using above all framework to run the testcases.  Could you help me out to tell me all framework need to use or not?

**Review by Ishticode — COMMENTED** [review]
The test has to be written once and it is automatically ran for all backends. Please check the details at [Ivy frontend tests](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html).

**Comment by SHUBHAPRIYA95** [comment]
Hi @Ishticode 
Thanks i got it.

**Comment by Ishticode** [comment]
This PR does not seem to add any meaningful contribution to the task referred in the title. The test cases have not been covered as requested with inactivity for over two months. Feel free to open if there is need. Thanks 🙂
